documentation/src/main/resources/pages/ditto/release_notes_200.md

```diff
@@ -3,29 +3,433 @@ title: Release notes 2.0.0
+
+
+#### Restructuring of Ditto's Maven modules
+
+Ditto's modules were adjusted to be structured in a more functional way. In Ditto 1.x the modules were structured
+in a more technical way.
+
+This table shows the old modules and in which module the old ones can be found in Ditto 2.0.0:
+
+| Ditto 1.x module                             | Ditto 2.x module          |
+| ---                                          | ---                       |
+| `ditto-model`                                | `-` (was pom only)        |
+| `- ditto-model-base`                         | `ditto-base/ditto-base-model` |
+| `- ditto-model-cleanup`                      | `-` (was internal API) |
+| `- ditto-model-connectivity`                 | `ditto-connections/ditto-connections-model` |
+| `- ditto-model-devops`                       | `ditto-devops/ditto-devops-model` |
+| `- ditto-model-enforcers`                    | `ditto-policies/ditto-policies-model` |
+| `- ditto-model-jwt`                          | `ditto-jwt/ditto-jwt-model` |
+| `- ditto-model-messages`                     | `ditto-messages/ditto-messages-model` |
+| `- ditto-model-namespaces`                   | `ditto-base/ditto-base-model` |
+| `- ditto-model-policies`                     | `ditto-policies/ditto-policies-model` |
+| `- ditto-model-query`                        | `ditto-rql/ditto-rql-query` |
+| `- ditto-model-rql`                          | `ditto-rql/ditto-rql-model` |
+| `- ditto-model-rql-parser`                   | `ditto-rql/ditto-rql-parser` |
+| `- ditto-model-things`                       | `ditto-things/ditto-things-model` |
+| `- ditto-model-thingsearch`                  | `ditto-thingsearch/ditto-thingsearch-model` |
+| `- ditto-model-thingsearch-parser`           | `ditto-rql/ditto-rql-parser` |
+| `ditto-protocol-adapter`                     | `ditto-protocol` |
+| `ditto-signals`                              | `-` (was pom only) |
+| `- ditto-signals-base`                       | `ditto-model-base` |
+| `- ditto-signals-acks`                       | `-` (was pom only) |
+| `-- ditto-signals-acks-base`                 | `ditto-base-model` |
+| `-- ditto-signals-acks-things`               | `ditto-things-model` |
+| `- ditto-signals-announcements`              | `-` (was pom only) |
+| `-- ditto-signals-announcements-base`        | `ditto-base-model` |
+| `-- ditto-signals-announcements-policies`    | `ditto-policies-model` |
+| `- ditto-signals-commands`                   | `-` (was pom only) |
+| `-- ditto-signals-commands-base`             | `ditto-base/ditto-base-model` |
+| `-- ditto-signals-commands-cleanup`          | `-` (was internal API) |
+| `-- ditto-signals-commands-common`           | `-` (was internal API) |
+| `-- ditto-signals-commands-connectivity`     | `ditto-connectivity/ditto-connectivity-model` |
+| `-- ditto-signals-commands-devops`           | `ditto-devops/ditto-devops-model` |
```

Comment:
ditto-model-devops was merged into ditto-base/ditto-base-api (org.eclipse.ditto.base.api.devops) and is considered to be internal API